test/integration/log_checker.go

```diff
@@ -153,7 +153,7 @@ func (lc *LogCheck) Execute() *LogCheck {
 		lc.firstCheck = now
 	}
 	r := regexp.MustCompile(`(?m)` + lc.searching)
-	logOptions := &corev1.PodLogOptions{}
+	var logOptions *corev1.PodLogOptions
```

Comment:
I don't get this change. I think we use the original declaration throughout our code base.

Reply:
the fact that its used throughout the codebase doesn't mean its right :) (both is used semi randomly in our code)
I personally prefer to use `var logOptions *corev1.PodLogOptions` because its just visually clearer for me

Reply:
Sure. It's matter of preference. :)

Reply:
It solves the `ineffectual assignments`. Let me try to explain it. Before this change we had:

`` logOptions := &corev1.PodLogOptions{}```

And right after that, we have an `if-else` statement, that will obligatorily instantiate a new value to `logOptions`, in this case, the first value will never be used, 'cause it will be replaced by the code inside the `if-else`. Did you get it? Is like, we define the variable, but do not initiate the struct, leaving it to the code inside the `if-else` statement.

Reply:
OK thanks for explanation.